packages/shared/src/lib/featureManagement.ts

```diff
@@ -22,6 +22,7 @@ const feature = {
   }),
   feedAdSpot: new Feature('feed_ad_spot', 0),
   searchVersion: new Feature('search_version', 2),
+  searchResultsUpgrade: new Feature('search_results_upgrade', false),
```

Comment:
Feature added in [GB](https://app.growthbook.io/features/search_results_upgrade).
@ilasw did you check the enrolment with Ido in the experiment channel already?

Reply:
Not already, was waiting for some reviews on PR first 👀

Reply:
Approved ✅